.devcontainer/devcontainer.json

```diff
@@ -1,43 +1,24 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
 {
-    "name": "PreTeXt-Full",
-    "features": {
-          "ghcr.io/rocker-org/devcontainer-features/pandoc:1": {}
-      },
-      "forwardPorts": [
-          27377,
-          27378
-      ],
-      "portsAttributes": {
-          "27378": {
-              "label": "CodeChat",
-              "onAutoForward": "openPreview",
-              "requireLocalPort": true,
-              "elevateIfNeeded": true,
-              "protocol": "https"
-          }
-      },
-    // Use 'postCreateCommand' to run commands after the container is created.
-    "postCreateCommand": "sudo bash ./.devcontainer/postCreateCommand.sh",
-  
-      // Configure tool-specific properties.
-      "customizations": {
-          "codespaces": {
-              "openFiles": [
-                  "project.ptx",
-                  "source/main.ptx"
-              ]
-          },
-          "vscode": {
-              "settings": {
-                  "CodeChat.CodeChatServer.Command": "CodeChat_Server"
-              },
-              "extensions": [
-                  "CodeChat.codechat",
-                  "oscarlevin.pretext-tools"
```

Comment:
It isn't crucial to have these, but doesn't hurt (and I get them anyway since I sync my vscode settings).  In particular, pretext-tools also gives snippet and syntax highlighting support, and some limited schema checking.